content/docs/06-integrations/00-cloudanix.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Cloudanix'
+metaTitle: 'cloudanix'
+metaDescription: 'Cloudanix container security helm chart for Spectro Cloud Palette'
+hiddenFromNav: true
+isIntegration: true
+category: ['security']
+logoUrl: 'https://cloudanix-assets.s3.amazonaws.com/static/cloudanix-logo-p.png'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Cloudanix
+
+Cloudanix container security helm chart detects anamolous behaviours on kubernetes cluster and provides an interactive interface in Cloudanix dashboard to view on which resources like pods, containers and nodes the activiy is occuring in real-time.
```

Comment:
after occuring, we should have a comma. Also, spelling mistake. `occurring, in real-time`